### PR TITLE
XEP-0369 misc editor fixes

### PR DESCRIPTION
**Pull request by linkmauve**
Sorry for having done a PR before the patch got merged, this is a rebase on top of current master.  Each commit is reviewable separately as a single unit.

### CODE REVIEW

File: xep-0369.xml [hunk 1]
```diff
@@ -637,9 +637,6 @@
     to='mix.shakespeare.example'
     from='hag66@shakespeare.example/pda'
     id='7nve413p'>
-  <register xmlns='urn:xmpp:mix:0'>
```
Comment (Kev) [inline xep-0369.xml]: > rfc6120 §8.2.3.6: “An IQ stanza of type "result" MUST include zero or one child elements.”

Well, yes, but this isn't a result stanza, it's an error.

Comment (SamWhited) [inline xep-0369.xml]: I feel that including the original payload is rather useless and we shouldn't encourage this behavior, so either way I think it's worth removing this.

**Comment by stevekille** [comment]
I have been over all of these, and from my perspective these are good to roll in,   Steve

File: xep-0369.xml (hunk 1 above)
Comment (linkmauve) [inline xep-0369.xml]: I will change the commit message, but the change should indeed stay.

Also github still randomly fails to send email notifications… /rant

Comment (Kev) [inline xep-0369.xml]: It's useful for eyeballing/debugging, but it's just a diagnostic thing, yes.

**Comment by Kev** [comment]
Let's merge at the same time as Sam's once Sam's have been reviewed, and we can do one version update with both.

**Pull request merged**